### DISCUSSION TITLE
Modal sizes

### DISCUSSION THREAD

**Discussion by namuller** [twilio-labs/paste#3507] [General] [answered]
Hi!
Does Modal only have 2 sizes "default" and "wide"? And for me to be able to edit Modal size is to use CustomizationProvider?

**Comment by serifluous** [comment]
Hey @namuller! Can you share a screenshot or prototype of your use case where you need a wider modal?

**Reply by namuller** [reply]
Right now I am using the size "wide". I want to make the Modal wider because of the placement of the DataGrid. 

<img width="1501" alt="Screenshot 2023-09-27 at 10 55 39" src="https://github.com/twilio-labs/paste/assets/86048435/37e8eb59-2a40-437f-9200-c1f3080f39c4">

**Reply by namuller** [reply, marked as answer]
But I guess the other option is to work with the DataGrid size and the size of its columns instead. 
I initially thought that there where more sizes to the Modal but I wasn't able to access them, but after further investigation I can see that there are 2 sizes for the Modal.
